openssl/src/dh.rs

```diff
@@ -1,3 +1,17 @@
+//! dh.rs
+//! 
+//! summary: bindings for dh library of OpenSSL
+//! author: steven fackler
+//! version: 
+//!
+//! dh is the diffie-hellman (dh) algorithm which allows for two parties to derive the key used for
+//! communication. in dh, the two parties agree to some large prime, `p`, and a generator, `g`,
+//! where 0 < g < p.
+//!
+//! For a full summary of the algo, please read https://wiki.openssl.org/index.php/Diffie_Hellman
+//! manpages: https://www.openssl.org/docs/manpages.html
```

Comment:
Why is this here?

Reply:
I felt bad for having this one link to a different source of documentation, the wiki, rather than the manpages as I use everywhere else in this file so I thought to directly link to the general manpage docs. I'll yank the line out.